scipy/sparse/array.py

```diff
@@ -0,0 +1,105 @@
+"""
+Sparse array creation functions e.g. `eye`, `diags`, etc.
+"""
+
+from . import _construct
+from ._arrays import (
```

Comment:
Probably worth hiding these from the `array` namespace so users don't get the array objects from here.

(You can use `del`, or override `__dir__`.)